### PR TITLE
Adding validation of timespan columns w/in 24hrs

### PR DESCRIPTION
**Pull request by benrr101**
Very small feature add to make sure that values entered for `TIME` columns are < 24hrs. If the value is >=24hrs when setting the column, an exception will be thrown that the edit/updateCell handler will catch and convert into an error on the JSON RPC side.

### CODE REVIEW

File: test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/CellUpdateTests.cs [hunk 1]
```diff
@@ -159,6 +159,17 @@ public void BoolInvalidFormatTest()
         }
 
         [Theory]
+        [InlineData("24:00:00")]
+        [InlineData("105:00:00")]
+        public void TimeSpanTooLargeTest(string value)
```
Comment (coquagli) [inline test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/CellUpdateTests.cs]: If you're feeling really ambitious you could make a test that verifies that timespans within the allowed range don't throw.

Comment (benrr101) [inline test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/CellUpdateTests.cs]: That's already there as part of the "RoundTripTest" right below this test 😃

Comment (coquagli) [inline test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/CellUpdateTests.cs]: Oops.

File: src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs [hunk 2]
```diff
@@ -196,6 +196,7 @@ private void ProcessBooleanCell(string valueAsString)
             ValueAsString = Value.ToString();
         }
 
+<<<<<<< HEAD
```
Comment (coquagli) [inline src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs]: Heads up for this.

**Pull request merged**